### PR TITLE
Made angle to encoder conversion

### PR DESCRIPTION
**Pull request by NWSchallock**
PTAL

### CODE REVIEW

File: src/main/cpp/subsystems/HatchIntake.cpp [hunk 1]
```diff
@@ -160,6 +162,21 @@ void HatchIntake::SetArmValue() {
     kPID_PrimaryClosedLoop, kTimeout_10Millis);
 }
 
+void HatchIntake::SetAngleValue() {
+  encoderFwd = mPivot.GetSelectedSensorPosition(kPID_PrimaryClosedLoop);
+  encoderRev = mPivot.GetSelectedSensorPosition(kPID_PrimaryClosedLoop);
```
Comment (duane701) [inline src/main/cpp/subsystems/HatchIntake.cpp]: `encoderFwd` and `encoderRev` will always be the same value. `encoderFwd` should be captured in a different command. 

The command that calls `GetArmValuesFwd()`can capture `encoderFwd`, then the command that calls `SetAngleValue()` can capture `encoderRev`. 

That is, don't capture `encoderFwd` here. Assume there's a previous value.

File: src/main/include/subsystems/HatchIntake.h [hunk 2]
```diff
@@ -29,6 +29,8 @@ class HatchIntake : public frc::Subsystem {
   int potRev;
   int encoderFwd;
   int encoderRev;
+  int angleFwd;
+  int angleRev;
```
Comment (duane701) [inline src/main/include/subsystems/HatchIntake.h]:
```
  int angleFwd;
  int angleRev;
```

Are never used. Remove these.

**Comment by duane701** [comment]
Also, remove the compiler guards in `HatchIntake.h`

**Comment by NWSchallock** [comment]
PTAL

File: src/main/cpp/subsystems/HatchIntake.cpp [hunk 3]
```diff
@@ -145,6 +145,9 @@ void HatchIntake::GetArmValuesFwd() {
 void HatchIntake::GetArmValuesRev() {
   potRev = armPot.GetValue();
   encoderRev = mPivot.GetSelectedSensorPosition(kPID_PrimaryClosedLoop);
+}
+
+void HatchIntake::SetEncoderValues() {
```
Comment (duane701) [inline src/main/cpp/subsystems/HatchIntake.cpp]: Need a command to call `SetEncoderValues()`

**Comment by NWSchallock** [comment]
PTAL

**Comment by duane701** [comment]
LGTM

Rebasing.

**Comment by duane701** [comment]
Replaced by #48

**Pull request closed without merging**